src/Organizer/CommandHandler/DeleteOrganizerHandler.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Organizer\CommandHandler;
+
+use Broadway\CommandHandling\CommandHandler;
+use CultuurNet\UDB3\Organizer\Commands\DeleteOrganizer;
+use CultuurNet\UDB3\Organizer\OrganizerRelationServiceInterface;
+use CultuurNet\UDB3\Organizer\OrganizerRepository;
+
+final class DeleteOrganizerHandler implements CommandHandler
+{
+    private OrganizerRepository $organizerRepository;
+    private OrganizerRelationServiceInterface $eventRelations;
+    private OrganizerRelationServiceInterface $placeRelations;
+
+    public function __construct(
+        OrganizerRepository $organizerRepository,
+        OrganizerRelationServiceInterface $eventRelations,
+        OrganizerRelationServiceInterface $placeRelations
+    ) {
+        $this->organizerRepository = $organizerRepository;
+        $this->eventRelations = $eventRelations;
+        $this->placeRelations = $placeRelations;
+    }
+
+    public function handle($command): void
+    {
+        if (!$command instanceof DeleteOrganizer) {
+            return;
+        }
+
+        $this->eventRelations->deleteOrganizer($command->getItemId());
+        $this->placeRelations->deleteOrganizer($command->getItemId());
```

Comment:
Shouldn't this be handled by a read model projector? The relations tables are read models no?

Reply:
Oh I see this uses the relation read models to find the related events/places and then dispatches `DeleteOrganizer` commands for them so it actually performs a write operation